### PR TITLE
[res] Add a TensorFlowLiteRecipes to test fold Shape

### PR DESCRIPTION
**Pull request by miusic**
This commit adds a TensorFlowLiteRecipes for testing to fold Shape

---
Related to: #12046
Draft: #12407

ONE-DCO-1.0-Signed-off-by: jihunnn-kim <jihunnn.kim@samsung.com>

### CODE REVIEW

**Comment by miusic** [comment]
This recipe will create the below model

model(before)

**Comment by miusic** [comment]
Below is the expected model after fold_shape is applied

model(after)

Test result is in the draft (#12407)

**Comment by seanshpark** [comment]
Use constant instead of ifm2

**Comment by miusic** [comment]
> Use constant instead of ifm2

Updated! 😄

---
This recipe will create the below model

model2(before)

And below is the expected model after fold_shape is applied

model2(after)

You can check the test result in the draft (#12407)

File: res/TensorFlowLiteRecipes/Net_Shape_Add_000/test.recipe [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+operand {
+  name: "ifm"
+  type: INT32
+  shape { dim: 4 }
+}
+operand {
+  name: "constant"
+  type: FLOAT32
+  shape { dim: 1 dim: 32 dim: 64 dim: 32 }
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Shape_Add_000/test.recipe:11]: if not needed with this size, can we use smaller vales like `[1, 2, 3, 4]`?

Comment (miusic) [reply res/TensorFlowLiteRecipes/Net_Shape_Add_000/test.recipe]: Ok, large size is not essential. Updated :)

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by miusic** [comment]
@mhs4670go May I ask you to take a look please? 🙂

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**